cat-launcher/src-tauri/src/variants/commands.rs

```diff
@@ -0,0 +1,38 @@
+use tauri::{command, State};
+use crate::variants::GameVariant;
+use crate::variants::get_game_variants_info::{self, GameVariantInfo};
+use crate::variants::repository::sqlite_game_variant_order_repository::SqliteGameVariantOrderRepository;
```

Comment:
Simplify the import path of SqliteGameVariantOrderRepository by reexporting it in the mod.rs file. See how it's done for other repositories.